src/index.js

```diff
@@ -92,6 +92,7 @@ export default class Swup {
 		this.updateTransition = updateTransition;
 		this.getAnimationPromises = getAnimationPromises;
 		this.getPageData = getPageData;
+		this.getAnchorElement = getAnchorElement;
```

Comment:
The naming doesn't make much sense, but I always used to separate modules vs. helpers as being available on Swup instance vs. not being available on Swup. I agree that this helper should be on Swup, but it's a bit breaking the convention this way, making things inconsistent. 
Should we move it under modules? Maybe even rename modules to something more fitting?

Reply:
Let's move it under modules and think of a better naming strategy later :)